### ISSUE TITLE
Unethical

### ISSUE THREAD

**Issue by robdekort** [WordPress/secure-custom-fields#5]
No description needed.

**Comment by lengthofrope** [comment]
I have the same issue

**Comment by sytheveenje** [comment]
Me too, please fix this asap. It's killing moral and the ecosystem.

**Comment by marcorieser** [comment]
Is this the same as ACF or am I missing something? I am confused. I was a ACF user for s long time and it upgraded to that plugin? Why?

**Comment by duanestorey** [comment]
Agreed. Hugely unethical. I wouldn’t want my name associated with this in perpetuity.

**Comment by avocadesign** [comment]
Agreed 100%, this goes against everything WP is supposed to stand for!

**Comment by efc** [comment]
Never been a fan of ACF, but this is beyond the pale.

**Comment by trishacupra** [comment]
I, too, keep having this `Unethical` error repeatedly with this plugin, and only this plugin. I believe it's an insecurity issue as well.

**Comment by febryanvaldo** [comment]
I recently got this `unethical error, please contact the actual developer of this plugin`. How to fix it?

**Comment by rickblackdog** [comment]
Saw this - think the newest grow the F up patch should solve it
